calour/__init__.py

```diff
@@ -8,3 +8,8 @@
 
 __credits__ = "https://github.com/biocore/calour/graphs/contributors"
 __version__ = "0.1.0.dev0"
+
+from calour.experiment import *
+from calour.io import *
+from calour.filtering import *
+from calour.sorting import *
```

Comment:
right :)
is there an elegant way to import all functions into the class (i.e. without explicitly writing each function name?)